tools/checked-c-convert/ProgramInfo.cpp

```diff
@@ -63,7 +66,7 @@ PointerVariableConstraint::PointerVariableConstraint(const Type *_Ty,
   BaseType = tyToStr(Ty);
 
   // Special case for void to not make _Ptr<void> pointers.
-  if( BaseType == "void" ) 
+  if( BaseType == "void" || BaseType == "struct __va_list_tag *" ) 
```

Comment:
There should be some way to test the type directly, instead of serializing it to a string.